conans/client/conan_api.py

```diff
@@ -233,7 +233,7 @@ def __init__(self, cache_folder=None, output=None, user_io=None, http_requester=
         self.runner = runner
         self.app = None  # Api calls will create a new one every call
         # Migration system
-        migrator = ClientMigrator(self.cache_folder, Version(client_version), self.out)
+        migrator = Migrator(self.cache_folder, Version(client_version), self.out)
```

Comment:
But better leave the ClientMigrator. It is going to be needed, migrations will be needed in future Conan 2.X versions, and the Migrator infra is also used in the server, so it would be dirty to put migrations in the base ``Migrator`` class. The class can remain, even if empty.

Reply:
Yes, it makes sense. I gonna revert it.